### DISCUSSION TITLE
Deploy>Contentplugins fails

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-Ed/Community#277] [Installation] [answered]
Hi Team,

CSP: GCP
Sunbirdevops- 5.1.0
I am trying to deploy a content plugin it fails
Build version -release-5.1.0

`TASK [delete and re-upload plugins] ********************************************
[WARNING]: Unable to find
'/var/lib/jenkins/workspace/Deploy/dev/Plugins/ContentPlugins/ansible/content-
plugins/plugins_to_delete_and_upload.txt' in expected paths (use -vvvvv to see
paths)`

**Comment by beepdot** [comment, marked as answer]
Please use https://github.com/project-sunbird/sunbird-content-plugins/tree/release-5.2.0 branch for the build job.
The `plugins_to_delete_and_upload.txt` was added as part of `release-5.2.0` branch.

**Reply by tsprasath** [reply]
@keshavprasadms  Deployment is successful but no content in the content-pluginGCP bucket.

`PLAY [local] *******************************************************************

TASK [set common azure variables] **********************************************
skipping: [localhost]

TASK [delete and re-upload plugins] ********************************************
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.activitybrowser-1.3']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.assessmentbrowser-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.assetbrowser-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.assetbrowser-1.2']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.assetbrowser-1.3']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.audio-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.audio-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.breadcrumb-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.ceheader-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.collaborator-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.collaborator-1.2']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.collectioneditor-1.5']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.collectioneditor-1.6']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.collectionheader-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.collectionkeyboardshortcuts-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.collectionwhatsnew-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.colorpicker-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.conceptselector-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.config-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.contenteditorfunctions-1.2']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.contentmeta-1.5']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.coursemeta-1.3']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.courseunitmeta-1.7']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.developer-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.download-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.editcontentmeta-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.editcontentmeta-1.2']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.genericeditor-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.genericeditor-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.genericeditorheader-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.genericeditorpreview-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.genericeditorsidebar-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.hotspot-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.image-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.iterator-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.keyboard-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.keyboardshortcuts-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.lessonbrowser-1.5']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.lessonplanmeta-1.3']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.lessonplanunitmeta-1.5']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.libs.ckeditor-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.mathfunction-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.mathtext-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.metadata-1.5']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.navigation-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.preview-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.preview-1.2']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.question-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionbank-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionset-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionset.preview-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionset.preview-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionset.quiz-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionunit-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionunit-1.2']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionunit.ftb-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionunit.ftb-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionunit.mcq-1.3']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionunit.mtf-1.2']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionunit.reorder-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionunit.reorder-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionunit.sequence-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.questionunit.sequence-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.quiz-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.readalongbrowser-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.review-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.reviewercomments-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.richtext-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.scribblepad-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.shape-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.stage-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.stageconfig-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.suggestcontent-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.summary-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.sunbirdcommonheader-1.9']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.sunbirdmetadata-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.telemetry-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.templatebrowser-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.text-1.2']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.textbookmeta-1.3']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.todo-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.topicselector-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.unitmeta-1.7']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.unsupported-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.uploadcontent-1.5']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.uploadfile-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.uploadlargecontent-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.utils-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.video-1.4']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.video-1.5']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.viewecml-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.wordinfobrowser-1.0']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.ekstep.wordinfobrowser-1.1']) 
skipping: [localhost] => (item=['blob-delete-batch-no-poll.yml', 'org.sunbird.questionunit.quml-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.activitybrowser-1.3']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.assessmentbrowser-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.assetbrowser-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.assetbrowser-1.2']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.assetbrowser-1.3']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.audio-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.audio-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.breadcrumb-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.ceheader-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.collaborator-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.collaborator-1.2']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.collectioneditor-1.5']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.collectioneditor-1.6']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.collectionheader-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.collectionkeyboardshortcuts-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.collectionwhatsnew-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.colorpicker-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.conceptselector-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.config-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.contenteditorfunctions-1.2']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.contentmeta-1.5']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.coursemeta-1.3']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.courseunitmeta-1.7']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.developer-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.download-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.editcontentmeta-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.editcontentmeta-1.2']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.genericeditor-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.genericeditor-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.genericeditorheader-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.genericeditorpreview-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.genericeditorsidebar-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.hotspot-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.image-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.iterator-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.keyboard-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.keyboardshortcuts-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.lessonbrowser-1.5']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.lessonplanmeta-1.3']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.lessonplanunitmeta-1.5']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.libs.ckeditor-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.mathfunction-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.mathtext-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.metadata-1.5']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.navigation-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.preview-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.preview-1.2']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.question-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionbank-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionset-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionset.preview-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionset.preview-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionset.quiz-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionunit-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionunit-1.2']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionunit.ftb-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionunit.ftb-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionunit.mcq-1.3']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionunit.mtf-1.2']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionunit.reorder-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionunit.reorder-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionunit.sequence-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.questionunit.sequence-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.quiz-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.readalongbrowser-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.review-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.reviewercomments-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.richtext-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.scribblepad-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.shape-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.stage-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.stageconfig-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.suggestcontent-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.summary-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.sunbirdcommonheader-1.9']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.sunbirdmetadata-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.telemetry-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.templatebrowser-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.text-1.2']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.textbookmeta-1.3']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.todo-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.topicselector-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.unitmeta-1.7']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.unsupported-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.uploadcontent-1.5']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.uploadfile-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.uploadlargecontent-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.utils-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.video-1.4']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.video-1.5']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.viewecml-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.wordinfobrowser-1.0']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.ekstep.wordinfobrowser-1.1']) 
skipping: [localhost] => (item=['blob-upload-batch-no-poll.yml', 'org.sunbird.questionunit.quml-1.1']) 

TASK [set common gcloud variables] *********************************************
ok: [localhost]

TASK [set common aws variables] ************************************************
skipping: [localhost]

TASK [run the s3_copy.sh script] ***********************************************
skipping: [localhost]

PLAY RECAP *********************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=0    skipped=4    rescued=0    ignored=0 `

**Reply by beepdot** [reply]
@santhosh-tg Can you please check this and respond?

**Reply by santhosh-tg** [reply]
@tsprasath we need to update the GCP role for this.  We are working on it and will inform once its fixed.

**Reply by santhosh-tg** [reply]
@tsprasath We have updated the role. Please try to deploy now.

**Reply by tsprasath** [reply]
@santhosh-tg  Deployment success.

@keshavprasadms please close this thread.
